nativelib/src/main/scala/java/lang/Object.scala

```diff
@@ -37,7 +39,17 @@ class _Object {
   @inline def __scala_## : scala.Int =
     __hashCode
 
-  protected def __clone(): _Object = ???
+  protected def __clone(): _Object = {
+    val typePtr: Ptr[Type] = getType(this)
+    val size               = (!typePtr).size
+    val clone              = GC.malloc_atomic(size)
```

Comment:
Just `GC.malloc`, not `GC.malloc_atomic`. Atomic means that object stores no pointers to other objects.